MykSuite/src/main/java/com/infomaniak/lib/myksuite/ui/screens/MyKSuiteDashboardScreen.kt

```diff
@@ -0,0 +1,238 @@
+/*
+ * Infomaniak Core - Android
+ * Copyright (C) 2025 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.lib.myksuite.ui.screens
+
+import android.content.res.Configuration
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.infomaniak.lib.myksuite.R
+import com.infomaniak.lib.myksuite.ui.components.MyKSuiteChip
+import com.infomaniak.lib.myksuite.ui.components.WeightOneSpacer
+import com.infomaniak.lib.myksuite.ui.screens.components.AppStorageQuotas
+import com.infomaniak.lib.myksuite.ui.screens.components.ExpendableActionItem
+import com.infomaniak.lib.myksuite.ui.screens.components.LimitedFunctionalities
+import com.infomaniak.lib.myksuite.ui.screens.components.UserAvatar
+import com.infomaniak.lib.myksuite.ui.theme.Dimens
+import com.infomaniak.lib.myksuite.ui.theme.Margin
+import com.infomaniak.lib.myksuite.ui.theme.MyKSuiteTheme
+
+@Composable
+fun MyKSuiteDashboardScreen(
+    userName: String,
+    avatarUri: String = "",
+    dailySendingLimit: () -> String,
+    onClose: () -> Unit = {},
+) {
+    Box(
+        Modifier
+            .verticalScroll(rememberScrollState())
```

Comment:
The whole screen is scrolling, including the topAppBar

Reply:
Also on tablet, the layout if very wide, but not sure there's much we can do about it